vms/platformvm/txs/executor/staker_tx_verification.go

```diff
@@ -762,3 +715,27 @@ func verifyTransferSubnetOwnershipTx(
 
 	return nil
 }
+
+// Ensure the proposed validator starts after the current time
+func checkStakerStartTime(chainTime, stakerTime time.Time) error {
+	if !chainTime.Before(stakerTime) {
+		return fmt.Errorf(
+			"%w: %s >= %s",
+			ErrTimestampNotBeforeStartTime,
+			chainTime,
+			stakerTime,
+		)
+	}
+	return nil
+}
+
+func stakerTooMuchIntheFuture(chainTime, stakerStartTime time.Time) error {
```

Comment:
nit: I think these functions should both start with `verify`. Maybe `verifyStakerTimeHasNotPassed` and `verifyStakerStartsSoon`

Reply:
done